### PR TITLE
:sparkles: Add searchText field

### PR DESCRIPTION
**Pull request by evans-g-crsj**

### CODE REVIEW

File: scripts/localsuperlinter
Comment (evans-g-crsj) [inline scripts/localsuperlinter:1]: Allows to run locally super linter. We can spare time and resources by testing locally before pushing a branch.

File: scripts/console [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-# scripts/console: Open a container instance of the applicaiton for testing/debugging
+# scripts/console: Open a container instance of the application for testing/debugging
 
 # Once in a terminal you can run, for instance: "black <my_python_file>" to reformat a file
-docker run --rm -it -v "$PWD":/code --workdir /code python:3.9-slim-buster sh -c "pip install -r src/requirements.txt -r requirements.txt && sh"
+docker run --rm -it -v "$PWD":/code --network=host --workdir /code python:3.9-slim-buster sh -c "pip install -r src/requirements.txt -r requirements.txt && sh"
```
Comment (evans-g-crsj) [inline scripts/console:6]: Need to communicate with remote clusters.

File: src/copy_member_to_members.py
Comment (evans-g-crsj) [inline src/copy_member_to_members.py:1]: Added step in the pipeline to create a new field `searchText` with legacy documents.

**Review by devbyaccident — APPROVED** [review]
Looks good to me

**Pull request merged**